plugins/microchip/switchtec-nvme-device.c

```diff
@@ -71,7 +71,7 @@ int pax_nvme_submit_admin_passthru_1k_rsp(int fd,
 	struct pax_nvme_device *pax;
 	struct switchtec_device_manage_nvme_req req;
 	struct switchtec_device_manage_nvme_rsp rsp;
-	int data_len;
+	size_t data_len = 0;
```

Comment:
Why initialize to zero? I see data_len is properly assigned before using.

Reply:
This is not absolutely necessary. Can be removed without any issue.